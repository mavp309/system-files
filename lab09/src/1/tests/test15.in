i 10
i 10
i 20
p -1
d 10
s 10
p -1
q 0
