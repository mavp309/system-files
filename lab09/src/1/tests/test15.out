20 10 10 
Deleted
True
20 10
